tests/test_main.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import sys
+from unittest.mock import patch, call
+
+import pytest
+
+sys.path.insert(0,'src')
+from main import main
+
+
+@patch("main.Controller")
+def test_main__argparse__default(patched_controller):
+    main([])
+    assert logging.root.level == 20
```

Comment:
Better to test these with the human-friendly constants:
```python
assert logging.root.level == logging.INFO
```

Reply:
done